Wireframe/index.html

```diff
@@ -1,33 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wireframe</title>
+     <meta name="viewport" content="width=device-width, initial-scale=0.0" />
+    <title>Web Development Basics</title>
+    <meta name="description" content="Learn about README files, wireframes, and Git branches in web development.">
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
+       <h1>Web Development Basics</h1>
+      <p>Web development is the process of creating websites and web applications for the internet and its involve front-end and back-end development.</p>
```

Comment:
In the given wireframe image, the header and description are centred on the page, but in your example, they are positioned on the left side of the page. Can you think of a way to change the layout of these texts?

The given wireframe:
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/0ddf132c-c26b-4f74-8f6e-14197c392c96" />

Your example:
<img width="1481" alt="image" src="https://github.com/user-attachments/assets/2708ac9a-cad8-4f92-a8f7-1f29991c4318" />